test/integration/migrate/index.js

```diff
@@ -268,12 +221,52 @@ module.exports = function(knex) {
 
   });
 
+  describe('knex.migrate.latest in parallel', function() {
+
+    afterEach(function() {
+      return knex.migrate.rollback({directory: 'test/integration/migrate/test'});
+    });
+
+    if (knex.client.dialect === 'postgres') {
+      it("is able to run two migrations in parallel in postgres", function () {
```

Comment:
Umm... I don't understand why auto-commit would be a problem here? If each migration is runnings  are done in their own transactions, auto-commit is automatically disabled in mysql? I'm a bit worried about the fact that this functionality ended up working only for postgresql.